get-resource.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash -xe
 #CACHEURL=http://172.22.0.1/images
 
+# Check and set http(s)_proxy. Required for cURL to use a proxy
+export http_proxy=${http_proxy:-$HTTP_PROXY}
+export https_proxy=${https_proxy:-$HTTPS_PROXY}
```

Comment:
The curl docs say the lowercase problem only applies to `http`: https://curl.haxx.se/libcurl/c/libcurl-env.htm, I guess it looks at HTTPS proxy but probably safe to just set both.

I wonder if we shouldn't also set no_proxy.

Reply:
It looks like curl should honor NO_PROXY (caps). https://curl.haxx.se/libcurl/c/libcurl-env.html

Reply:
When I tested locally, curl required lowercase variable for https as well. NO_PROXY should work, I think.